recipes/moments/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+cd $SRC_DIR/moments
```

Comment:
When Conda download the src archive, it also `cd`in it.
So you can consider in the build.sh that you are already in the src directory.
You can remove this line.